tests/test_states.py

```diff
@@ -220,6 +247,394 @@ def test_squeezed_coherent(self):
         self.assertAllAlmostEqual(res.flatten(), res_exact.flatten(), delta=self.tol)
 
 
+class PolyQuadExpectationSingleMode(BaseTest):
+    num_subsystems = 3
+    cutoff = 12
```

Comment:
We should prefer not to have one of the user-adjustable test parameters explicitly bypassed in the tests (at least, not without warning).

Reply:
I originally tried that; unfortunately, a cutoff of 5 (the default) causes the expectation of the quadratic terms to have too much error, causing the assertions to fail

Reply:
We should either change the parameters of the test to fit within that cutoff, or add a warning msg inside the test that the user-provided value is overridden (if it is lower than 12)

Reply:
I added a logger warning to the relevant tests